packages/internals/src/utils/__tests__/isInteractive.test.ts

```diff
@@ -0,0 +1,40 @@
+import { isInteractive } from '../isInteractive'
+
+const originalEnv = { ...process.env }
+
+describe('isInteractive', () => {
+  beforeEach(() => {
+    process.env = { ...originalEnv }
+  })
+  afterAll(() => {
+    process.env = { ...originalEnv }
+  })
+
+  describe('in non TTY environment', () => {
+    const mockedValue = { isTTY: false }
+    test('isInteractive should be false', () => {
+      // @ts-ignore
+      expect(isInteractive({ stream: mockedValue })).toBe(false)
```

Comment:
Do we really want to avoid typing the `isInteractive()` argument?

Reply:
What do you mean?

Reply:
Ah the `@ts-ignore`

Reply:
fixed